linux_files/setup

```diff
@@ -18,7 +18,7 @@ echo
 }
 
 function continueprompt {
-read -r -p "Would you like to continue with this setup script? [y/N] " response
+read -r -p "Would you like to continue with this setup script? [y/N] " response         # this says it defaults to N but when I hit enter to exit it continues the script anyways
 if [[ "$response" =~ ^[yY]$ ]] || [[ -z $response ]]
```

Comment:
I think this is my mistake when I am editing... it suppose to be Y/n

Reply:
Gotcha. Thank you.